### PR TITLE
Add About Regional Plan Section to Home page.

### LINKED ISSUE #222: Add About Regional Plan Section
**Describe the feature. What should a user be able to do?**
This section will be part of the home page rather than a separate about page.  The "learn more" button should link to the MAPC video (https://youtu.be/bApghSqlbYc).  At the smallest screen size (500px) the triangle element should disappear. 

From 3.0 XD file:
screen shot 2019-03-06 at 4 25 54 pm

**Why do we need this feature?**
This is needed as part of the Phase 2 update to align with other style changes.  Will give more content to home page rather than creating separate about.

**Alternatives**
@Kit-Un provide any suggestions you have for including a triangle element for mobile views rather than removing.

### PR DESCRIPTION
**Pull request by smithWEBtek**
* Used image, text and design from AdobeXD file.
* Added CSS classes required for the design.
* Added media query for image to disappear at 500px screen width.
* The Learn button navigates to the required YouTube video, in a separate browser window.

Resolves #222 .

# Why is this change necessary?
This conforms to the design request to have the home page include the content that would otherwise be seen at the About page.

# How does it address the issue?
The code changes the CSS properties of the home page, to conform to the design requirements.

# What side effects does it have?
No new side effects. Existing CSS classes where used as much as possible, including existing variables.

### CODE REVIEW

File: app/views/refinery/pages/home.html.erb [hunk 1]
```diff
@@ -5,50 +5,46 @@
...
-          <a class="button" href="<%= CONSTANT_CONTACT_LANDING_PAGE %>" target="_blank"><%= t('refinery.home.email_sign_up.join') %></a>
+        <div class="content-column-one--regional-plan-text">
+          <h1 class="content-column-two--regional-plan-text content-column-two--regional-plan-title">What is a Regional Plan?</h1>
+          <p>It’s a set of actions that local and state government can take to improve the region over time. Plans
+            like MetroCommon can include thinking about many different aspects of our lives. That’s because policies and laws can affect jobs, transportation, health, and more.</p>
+            <p>MetroCommon will focus on challenges and opportunities that
+            are bigger than any one place. And it will look at how those issues are connected. Topics like traffic
+            congestion, the rising cost of housing, our changing climate, equity of wealth and health, and efficient government – all will be part of the mix.    
+            </p>
+          <a class="content-column-two__button--learn" href="https://youtu.be/bApghSqlbYc" target="_blank">LEARN</a>
```
Comment (mzagaja) [inline app/views/refinery/pages/home.html.erb]: What about making this `Learn More` to conform with the design suggestion:
image

and then use `text-transform: uppercase;` as a style for `content-column-two__button--learn`

Also on my browser font-family is loading as SwiftNeueLTPro, and the template suggests the button should be Calibre Semibold.

Comment (mzagaja) [reply app/views/refinery/pages/home.html.erb]: Also would you consider adding a little padding/margin to the top of the button to separate it from the paragraph?

File: app/views/refinery/pages/home.html.erb [hunk 2]
```diff
@@ -5,50 +5,46 @@
         <h1><%= @page.title %></h1>
         <%= raw @page.content_for(:subtitle) %>
         <% if Flipper.enabled?(:search) %>
-          <%= render "/refinery/shared/search" %>
+        <%= render "/refinery/shared/search" %>
```
[the remark this answers is not in the document]
Comment (smithWEBtek) [reply app/views/refinery/pages/home.html.erb]: No reason, must have been an oversight, or maybe a linter did it when I formatted the page as a .html doc.  I've put the indent back in and will watch for linting changes like this moving forward.

File: app/views/refinery/pages/home.html.erb (hunk 1 above)
Comment (smithWEBtek) [reply app/views/refinery/pages/home.html.erb]: I've added divs in `home.html.erb` and  css classes in `home.scss` to separate the button and its text, allowing the button to have top padding. 

I see what happened now with the "LEARN MORE" button.  The file shows just the word "LEARN" until you expand the button selector handle a little bit to the right, so now I see the full text intended for the button.  I've updated the text and implemented your suggestions.

**Comment by mzagaja** [comment]
Superseded by #243

**Pull request closed without merging**